stories/getting-started/installation.mdx

```diff
@@ -152,7 +152,25 @@ Add this into the `head` of your html file:
     type="text/css"
     href="https://assets.life.gov.sg/react-design-system/v3/css/atkinson-hyperlegible-next.css"
 />
-```
+
```

Comment:
hmm the formatting here seems to be broken, the html code for PA needs to be enclosed

and is the four backticks intentional?